sql3/test/helpers.go

```diff
@@ -14,14 +14,23 @@ import (
 )
 
 // MustQueryRows returns the row results as a slice of []interface{}, along with the columns, the query plan as a []byte or an error.
-func MustQueryRows(tb testing.TB, svr *featurebase.Server, q string) ([][]interface{}, []*featurebase.WireQueryField, []byte, error) {
+func MustQueryRows(tb testing.TB, svr *featurebase.Server, c context.Context, q string) ([][]interface{}, []*featurebase.WireQueryField, []byte, error) {
```

Comment:
I... have bad news.

There's a fairly strong idiom that ctx parameters Come First. There's also a strong idiom that testing.TB parameters Come First. But I think that, for maintainence-is-easier reasons, we should move the context in front of the server here.

I'm aware this requires a large search and replace elsewhere, sorry.